content/getting-started/accounts.md

```diff
@@ -0,0 +1,19 @@
+---
+menu:
+  main:
+    parent: getting-started
+title: Accounts
+weight: -100
```

Comment:
Pretty sure I just copied this from another page. I've only been adjusting the `weight`s as necessary.